doc/tutorials/QuickAddressBook/QuickAddressBookTypes/AddressBookItem.qml

```diff
@@ -17,8 +16,7 @@ Rectangle {
 
     RowLayout {
         spacing: 12
-        anchors.left: parent.left
-        anchors.leftMargin: spacing
+        Layout.fillWidth: true
         RoundButton {
             id: deleteButton
             text: "🗙"
```

Comment:
I can't figure out how to do this better. But you need to qualify the lookups on line 30 to this:
`text: "<b>" + addressBookItem.name + "</b><br><i>" + addressBookItem.addr + "</i>"`